.eslintrc.js

```diff
@@ -0,0 +1,85 @@
+module.exports = {
+  root: true,
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: "module",
+  },
+  env: {
+    node: true,
+    es6: true,
+  },
+  settings: {
+    react: {
+      version: "detect",
+    },
+  },
+  plugins: ["prettier"],
+  extends: ["eslint:recommended", "prettier"],
+  rules: {
+    "prettier/prettier": "error",
+  },
+  overrides: [
+    {
+      files: ["**/*.{ts,tsx}"],
+      parser: "@typescript-eslint/parser",
+      parserOptions: {
+        project: "tsconfig.json",
+      },
+      env: {
+        browser: true,
+        node: true,
+      },
+      plugins: [
+        "@typescript-eslint",
+        "jsx-a11y",
+        "simple-import-sort",
+        "react",
+      ],
+      extends: [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:react/recommended",
+        "plugin:react-hooks/recommended",
+        "plugin:jsx-a11y/recommended",
+        "next",
+        "prettier",
+      ],
+      rules: {
+        "prettier/prettier": "error",
+
+        // Relax some TypeScript rules to make them more accessible to beginners.
+        "@typescript-eslint/ban-ts-comment": "off",
+        "@typescript-eslint/no-explicit-any": "off",
+        "@typescript-eslint/no-non-null-assertion": "off",
+        "@typescript-eslint/no-unsafe-assignment": "off",
+
+        // Sort imports.
+        "simple-import-sort/imports": [
+          "error",
+          {
+            groups: [
+              ["^\\u0000"], // Side effect imports.
+              ["^next", "^react", "^@?\\w"], // Packages. Put `next`/`react`-related packages first.
+              ["^@/(components|lib)(/.*|$)"], // Internal paths - change these to match your project structure defined in tsconfig.json.
+              ["^\\.\\.(?!/?$)", "^\\.\\./?$"], // Parent imports. Put `..` last.
+              ["^\\./(?=.*/)(?!/?$)", "^\\.(?!/?$)", "^\\./?$"], // Other relative imports. Put same-folder imports and `.` last.
+            ],
+          },
+        ],
+        "simple-import-sort/exports": "error",
+
+        // Allow unused variables that start with an underscore.
```

Comment:
Is there some specific use case for this?

Reply:
Not sure which line of code this question refers to?

If it's about `"simple-import-sort/exports": "error",` then I agree it's much less valuable than import sorting. Details are [here](https://github.com/lydell/eslint-plugin-simple-import-sort#exports) but I'm happy to get rid of this rule if we don't see a use for it.

Or if it's about `Allow unused variables that start with an underscore`, then I often find it's useful to include *all* arguments that are available in some function you don't control. Even if those arguments aren't used quite yet, it can be useful to not forget that they're there. Of course we can also remove it if that's just me, though.

Reply:
Ah, I was referencing the line 71. Didn't notice that the highlight reached those other lines, sorry!

I was just curious what I am doing differently as I haven't had this issue in a long time. I remember that when working with Express it's often the case that you need to skip some of the parameters with the `_`, but I feel that many libraries have moved to some sort of config object instead of multiple parameters.

I mean, you rarely see `someFunction(param1, param2, param3)` but `someFunction(options)` where options is an object. And I would suggest that (latter) style for own helpers too, as they are easier to refactor and so on.

But after all, I don't have opinion in either direction with this rule 👍